volumes/volume.go

```diff
@@ -90,8 +90,13 @@ func (v *Volume) initialize() error {
 	v.lock.Lock()
 	defer v.lock.Unlock()
 
-	if _, err := os.Stat(v.Path); err != nil && os.IsNotExist(err) {
-		if err := os.MkdirAll(v.Path, 0755); err != nil {
+	if _, err := os.Stat(v.Path); err != nil {
+		switch {
```

Comment:
I thought to use `switch` in order to chain future error messages in there and not with many `if`s. Alright btw :+1: